### DISCUSSION TITLE
Welcome to retroReversing Discussions!

### DISCUSSION THREAD

**Discussion by RetroGameDeveloper** [RetroReversing/retroReversing#45] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by perfectGenius** [comment]
+1 PS2 game with debug symbols - **True Crime** Streets of L.A. - [in module called STMMAN.IRX](https://www.patreon.com/posts/sins-of-ps2-true-34283439)
